expr/func_absolute.go

```diff
@@ -36,7 +35,7 @@ func (s *FuncAbsolute) Exec(dataMap DataMap) ([]models.Series, error) {
 		serie.Target = fmt.Sprintf("absolute(%s)", serie.Target)
 		serie.Tags = serie.CopyTagsWith("absolute", "1")
 		serie.QueryPatt = fmt.Sprintf("absolute(%s)", serie.QueryPatt)
-		out := pointSlicePool.Get().([]schema.Point)
+		out := pointSlicePool.Get()
```

Comment:
A lot of functions now how many datapoints they need and could use `GetMin`

Reply:
true, but i don't have time now to go over all those